internal/viridian/custom_class.go

```diff
@@ -0,0 +1,107 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+)
+
+func (a API) ListCustomClasses(ctx context.Context, cluster string) ([]CustomClass, error) {
+	cID, err := a.findClusterID(ctx, cluster)
+	if err != nil {
+		return nil, err
+	}
+	csw, err := doGet[[]CustomClass](ctx, fmt.Sprintf("/cluster/%s/custom_classes", cID), a.Token())
+	if err != nil {
+		return nil, fmt.Errorf("listing custom classes: %w", err)
+	}
+	return csw, nil
+}
+
+func (a API) UploadCustomClasses(ctx context.Context, sp clc.Spinner, cluster, filePath string) error {
```

Comment:
Let's not pass the spinner to this method in order to not depend on it.
Same comment for other `API` methods that takes a spinner.